code/game/objects/items/explosives/grenades/marines.dm

```diff
@@ -527,6 +527,11 @@
 		set_light_on(FALSE)
 
 /obj/item/explosive/grenade/flare/throw_impact(atom/hit_atom, speed)
+	if(isopenturf(hit_atom))
```

Comment:
Don't see a case where a node would be outside an open turf?

Reply:
needs to check for open turf as its happening before calling parent.